LATAM : Add channels + ALL : Cleaning

BellezaEmporium opened the pull request:
Astra streams, following Dum's MTV 80's addition.

BellezaEmporium committed:
CR : Add Repretel 11

Astra stream

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Update cr.m3u

BellezaEmporium committed:
Update hn.m3u

BellezaEmporium committed:
Update uk.m3u

Dum4G commented:
Please finish cleaning at least in a bot branch. We have 3 cleaning branches atm, none is finished.

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G commented:
Please check the US playlist for plex wurl links. AMC Weddings from them is not working for me

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Update es.m3u

BellezaEmporium committed:
Update es.m3u

Dum4G commented on channels/es.m3u:
```diff
@@ -83,8 +83,10 @@ https://cls.alcarria.tv/live/alcarriatv-livestream.m3u8
 #EXTINF:-1 tvg-id="AragonTV.es" tvg-name="Aragón TV" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/d/d4/Logo_aragon_tv_2016.png" group-title="",Aragón TV (720p)
 https://cartv.streaming.aranova.es/hls/live/aragontv_canal1.m3u8
-#EXTINF:-1 tvg-id="AtresSeriesInternacional.es" tvg-name="Atres Series Internacional" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="",Atres Series Internacional
+#EXTINF:-1 tvg-id="AtreseriesInternacional.es" tvg-name="Atreseries Internacional" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="",Atreseries Internacional
```
Internacional but Spanish language and Spain as a country? I'm confused

LaneSh4d0w replied:
Atreseries Internacional is a subsidiary channel of the Atresmedia Group, which is Spanish-based. That's why it's in the Spanish section.

https://www.lyngsat.com/tvchannels/es/Atreseries.html

LaneSh4d0w replied:
To be honest, the "Internacional" mention is only from the hashtag they put over their programs "#A3SeriesInt"

BellezaEmporium committed:
Update cr.m3u

BellezaEmporium committed:
Update es.m3u

BellezaEmporium committed:
Update mx.m3u

BellezaEmporium committed:
Update mx.m3u

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Update ni.m3u

BellezaEmporium committed:
DO : Add CDN + CDN Deportes

They do stream their program from their own website, but it would need some extra work to make it functional (tokened Akamai stream)

BellezaEmporium committed:
Update hn.m3u

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
ES : Re-correct A3S

Lyngsat indicates that it's Atreseries. Removing the "Internacional" mention.

BellezaEmporium committed:
Add more channels

BellezaEmporium committed:
Merge branch 'LaneSh4d0w-patch-1' of https://github.com/iptv-org/iptv into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
CO : Add tags, clean and add Cali TV

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Update co.m3u

BellezaEmporium committed:
Update co.m3u

BellezaEmporium committed:
CO : fix mistake + add more channels

Happy July 20th to all Colombians who use our playlists.

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Update es.m3u

BellezaEmporium committed:
ES : Add a few backup links

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Update co.m3u

BellezaEmporium committed:
Update cr.m3u

BellezaEmporium committed:
Update cr.m3u

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Update es.m3u

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Big clean

BellezaEmporium committed:
Quack

LaneSh4d0w commented:
Reason for the cleaning in US : 

- XUMO links that could be shortened
- Chunklist links and dupes while a playlist file existed.
- I kept only one channel (MTV Nordic) with his dupe 3500.m3u8, to see if the master was geo-locked or not.

BellezaEmporium committed:
This might mark the end of US clean

LaneSh4d0w commented:
Will be good for now. New PR soon for more channels, but this big US wash-up will make some good to the playlists.

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G commented on channels/us.m3u:
```diff
@@ -1874,12 +1852,6 @@ https://dai.google.com/linear/hls/event/EXltT2IOQvCIn8v23_15ow/master.m3u8
-#EXTINF:-1 tvg-id="dbd25a" tvg-name="IFC_Films_Picks" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/XJaSjhm.png" group-title="",IFC Films Picks
-https://amc-ifc-films-picks-1.plex.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="IFCFilmsPicks.us" tvg-name="IFC Films Picks" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/XJaSjhm.png" group-title="Movies",IFC Films Picks
-https://amc-ifc-films-picks-1.vizio.wurl.com/manifest/1200.m3u8
```
works

LaneSh4d0w replied:
Dupe of "AMC Films Picks by IFC"

Dum4G commented:
Both MTV Nordics do not work for me, not even with VPN

Dum4G commented on channels/us.m3u:
```diff
@@ -6648,10 +6438,6 @@ https://gideommd.mmdlive.lldns.net/gideommd/d5b6a753f86e4e51b8e15c7d23887a4f/pla
 #EXTINF:-1 tvg-id="YUYUTV.us" tvg-name="YUYU TV" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/Bbvtcd7.png" group-title="",YUYU TV
 https://yuyu-samsung.plex.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living (720p)
-https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8
```
works

LaneSh4d0w replied:
Already available on the Indian playlist.

Dum4G replied:
It's wrong then
https://en.wikipedia.org/wiki/Z_Living

LaneSh4d0w replied:
https://www.lyngsat.com/tvchannels/in/Z-Living-USA.html

Dum4G replied:
they're wrong :)

Dum4G commented on channels/us.m3u:
```diff
@@ -6648,10 +6438,6 @@ https://gideommd.mmdlive.lldns.net/gideommd/d5b6a753f86e4e51b8e15c7d23887a4f/pla
-#EXTINF:-1 tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living (720p)
-https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8
-#EXTINF:-1 tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living
-https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r_prod/index.m3u8
```
works

LaneSh4d0w replied:
Already available on the Indian playlist.

LaneSh4d0w commented:
Noted, i'm gonna see if I can get a working link.

Le jeu. 22 juil. 2021 à 20:38, Dum4G ***@***.***> a écrit :

> *@Dum4G* requested changes on this pull request.
> ------------------------------
>
> In channels/us.m3u
> <https://github.com/iptv-org/iptv/pull/3543#discussion_r675072264>:
>
> > @@ -6648,10 +6438,6 @@ https://gideommd.mmdlive.lldns.net/gideommd/d5b6a753f86e4e51b8e15c7d23887a4f/pla
>  https://thegateway.app/YouToo/CueTones/playlist.m3u8
>  #EXTINF:-1 tvg-id="YUYUTV.us" tvg-name="YUYU TV" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/Bbvtcd7.png" group-title="",YUYU TV
>  https://yuyu-samsung.plex.wurl.com/manifest/playlist.m3u8
> -#EXTINF:-1 <https://yuyu-samsung.plex.wurl.com/manifest/playlist.m3u8-#EXTINF:-1> tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living (720p)
> -https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8
>
> works
> ------------------------------
>
> In channels/us.m3u
> <https://github.com/iptv-org/iptv/pull/3543#discussion_r675072305>:
>
> > @@ -6648,10 +6438,6 @@ https://gideommd.mmdlive.lldns.net/gideommd/d5b6a753f86e4e51b8e15c7d23887a4f/pla
>  https://thegateway.app/YouToo/CueTones/playlist.m3u8
>  #EXTINF:-1 tvg-id="YUYUTV.us" tvg-name="YUYU TV" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/Bbvtcd7.png" group-title="",YUYU TV
>  https://yuyu-samsung.plex.wurl.com/manifest/playlist.m3u8
> -#EXTINF:-1 <https://yuyu-samsung.plex.wurl.com/manifest/playlist.m3u8-#EXTINF:-1> tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living (720p)
> -https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8
> -#EXTINF:-1 <https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8-#EXTINF:-1> tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living
> -https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r_prod/index.m3u8
>
> works
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/iptv-org/iptv/pull/3543#pullrequestreview-713119036>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHMM3ZMRSSWIBMW2CYI5JFTTZBQQ5ANCNFSM5AN3WT6A>
> .
>

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Deleted MTV Nordic

Depending on the "MTV, if it's the music channel, it closed on 2005"

BellezaEmporium committed:
Moving Z Living channels
